libs/dh/shared/test-util-metering-point/src/lib/dh-transloco-testing.module.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  TranslocoTestingModule,
+  TranslocoTestingOptions,
+} from '@ngneat/transloco';
+
+import {
+  da,
+  en,
+} from '@energinet-datahub/dh/globalization/assets-localization';
+
+export function getTranslocoTestingModule(
+  options: TranslocoTestingOptions = {}
+) {
+  return TranslocoTestingModule.forRoot({
+    langs: { da, en },
+    translocoConfig: {
+      availableLangs: ['da', 'en'],
+      defaultLang: 'da',
+    },
```

Comment:
suggestion: Ideally we can use `DisplayLanguage` enum instead of hardcoded language values (similar to `dh-transloco.module.ts`). But this also means adjusting the dependency rules.

suggestion: Lets use English as default language so we don't have to use Danish text in tests (if necessary).

Reply:
Totally agree with the enum, however in the test we can totally avoid using any kind of strings since we can just use the json :)

Reply:
But ofc. this is a testing principle we should talk about. Are we testing the text is correct or are we just testing the right translation Keys are used? 
@dzhavat @LayZeeDK

Reply:
Eventually, we might not be in control of the translation texts in the codebase if they are located in a database and managed by a different system. Event before then, non-developers might want to create pull requests for translation texts which would break tests if we query for the translation texts.

If we don't decide to go down this route, then:
A. How do we display the translation key instead of the translation text?
B. We should consider whether to verify availability of all translation keys in both display languages.

Reply:
I vote to us `DisplayLanguage` instead of hard-coded magic strings.

Reply:
As far as I understood it, we should be able to query elements using their text value. This makes the query specific but also makes sure that there is, in fact, text with the desired value on the page. So we're not testing translation keys. Do you have another take on this?

Reply:
I just saw Lars' comment above. When I take the whole translation management part into consideration, it makes sense to not rely on translation text in tests. This leaves a whole bunch of methods out so we should use other ways for querying elements.

Reply:
Lets talk about this in a call instead when I've changed the other stuff